debuginfo/src/object.rs

```diff
@@ -231,15 +231,15 @@ impl<'bytes> FatObject<'bytes> {
             FatObjectKind::Breakpad(_) => 1,
             FatObjectKind::Elf(..) => 1,
             FatObjectKind::MachO(ref mach) => match *mach {
-                mach::Mach::Fat(ref fat) => fat.iter_arches().count(),
+                mach::Mach::Fat(ref fat) => fat.narches,
```

Comment:
I wonder why we did that.

Reply:
Maybe it's new that this is publicly exposed. Or we just missed it.

Reply:
FWIW, he could also implement `size_hint` and `count` on the iterator, so that it doesn't matter.